<h2>Introduction</h2>
<br>
Single-cell RNA-seq data only offers static snapshots of data, since all cells are destroyed upon measurement. Thus it is impossible to monitor how an individual cell changes over time. 

Trajectory Net establishes a link between continuous normalizing flows (CNF) and dynamic optimal transport. Thus the transport problem can be efficiently solved using a Neural ODE framework. It controls the continuous paths taken between distributions to produce dynamic optimal transport
<br>
<strong >By using TrajectoryNet we can: </strong>
<li>Interpolate non-linearly using information from more than two timepoints</li>
<li>Interpolate by following the manifold of observed entities between measured time points, thereby solving the static-snapshot problem. </li>
<li>Create continuous-time trajectories of individual entities, giving researchers the ability to follow an entity in time. </li>
<li>Form a deep representational model of system dynamics, which can then be used to understand drivers of dynamics (gene logic in the cellular context), via perturbation of this deep model.</li>
<br>
The paper can be viewed on <a href="https://arxiv.org/abs/2002.04461"> Trajectory Net website</a>